### DISCUSSION TITLE
provision:npm fails on Azure VPS

### DISCUSSION THREAD

**Discussion by matsn0w** [deployphp/deployer#3135] [Help needed]
Hey there,

I noticed that provisioning an Ubuntu 20.04 VPS hosted on Azure fails everytime with this error:

```bash
task provision:npm
[production]  error  in nodejs.php on line 9:
[production] run kill -9 -
[production] err bash: line 1: kill: -: arguments must be process or job IDs
[production] exit code 1 (General error)
```

Any ideas why this is happening? This problem did not occur on a server hosted elsewhere. I tried two clean installs, both on Azure, and both have this issue.

**Comment by antonmedv** [comment]
Looks like pid on cmd was not found. So kill has nothing.

**Reply by matsn0w** [reply]
Yeah, that's obvious. The question is why :)

**Reply by antonmedv** [reply]
Command for grepping pid failed? ¯\\\_(ツ)\_/¯
